indico/modules/events/registration/client/js/form/fields/BooleanInput.jsx

```diff
@@ -91,14 +91,16 @@ export default function BooleanInput({
   placesLimit,
   placesUsed,
 }) {
+  defaultValue = defaultValue === '' ? null : defaultValue === 'yes';
```

Comment:
```
defaultValue = {yes: true, no: false}[defaultValue] || null
```

What about this? It would cover he case where we got crappy translated strings in the DB more nicely since it'd default to no value for those, instead of defaulting to "no"